src/components/PostsList.tsx

```diff
@@ -2,53 +2,66 @@
 
 import classNames from 'classnames';
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
 import { Post } from '../types/Post';
+import * as selectedPostActions from '../features/selectedPost';
+// import { Post } from '../types/Post';
 
-type Props = {
-  posts: Post[];
-  selectedPostId?: number;
-  onPostSelected: (post: Post | null) => void;
-};
+// type Props = {
+//   posts: Post[];
+//   selectedPostId?: number;
+//   onPostSelected: (post: Post | null) => void;
+// };
+
+export const PostsList: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const { selectedPost } = useAppSelector(state => state.selectedPost);
+  const { posts } = useAppSelector(state => state.posts);
 
-export const PostsList: React.FC<Props> = ({
-  posts,
-  selectedPostId = 0,
-  onPostSelected,
-}) => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+  const handleselectedPost = (post: Post) => {
```

Comment:
The function name `handleselectedPost` should be `handleSelectedPost` to follow camelCase naming conventions.